site/content/docs/developing/storage.md

```diff
@@ -30,6 +30,16 @@ $ copilot storage init -n users -t DynamoDB -w api --partition-key id:N --sort-k
 
 This will create a DynamoDB table called `${app}-${env}-${svc}-users`. Its partition key will be `id`, a `Number` attribute; its sort key will be `email`, a `String` attribute; and it will have a [local secondary index](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/LSI.html) (essentially an alternate sort key) on the `Number` attribute `post-count`. 
 
+It is also possible to create an RDS Aurora Serverless(Daiya: add link here mmkay?) cluster using `copilot storage init`. 
```

Comment:
🤔

Reply:
😭 😭  this part was written way too long ago i forgot to change it